docs/notebooks/failure_ego.pct.py

```diff
@@ -105,38 +104,29 @@ def create_classification_model(data):
     gpflow.set_trainable(vgp.kernel.variance, False)
     return vgp
 
-
 regression_model = create_regression_model(initial_data[OBJECTIVE])
 classification_model = create_classification_model(initial_data[FAILURE])
 
+
 # %% [markdown]
 # ## Build Trieste models
 #
 # We now specify how Trieste will use our GPflow models within the BO loop.
 #
-# For our `GPR` model, we will use a standard L-BFGS optimizer from Scipy, whereas we will optimze our `VGP` model using alternate Adam steps (to optimize kernel parameter) and NatGrad steps (to optimize variational parameters).
+# For our `VGP` model we will use a non-default optimization: alternate Adam steps (to optimize kernel parameter) and NatGrad steps (to optimize variational parameters). For this we need to use the `BatchOptimizer` wrapper and set the `use_natgrads` model argument to `True` in our `VariationalGaussianProcess` model wrapper.
```

Comment:
so is this notebook currently only working because we use a custom acquisition function that's not compiled (which means it doesn't hit the bug I'm trying to resolve)?

Reply:
probably